david.md

```diff
@@ -0,0 +1,3 @@
+collaberating with other engineers to solve complicated problems
```

Comment:
not always true, can be done individually

Reply:
collaboration seems to be for bigger ideas vs the actual work itself